### PR TITLE
Add initial GitHub Actions CI

### PR DESCRIPTION
**Pull request by tianon**
See docker-library/bashbrew#2. :+1:

(This is currently failing due to https://bugs.php.net/bug.php?id=79445, but I wanted to get it open to have a place to record the link to that bug. :sweat_smile:)

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -58,8 +57,8 @@ for version in "${versions[@]}"; do
 			.[$version].source[]
 			| select(.filename | endswith(".xz"))
 			|
-				"https://www.php.net/get/" + .filename + "/from/this/mirror",
-				"https://www.php.net/get/" + .filename + ".asc/from/this/mirror",
+				"https://www.php.net/distributions/" + .filename,
+				"https://www.php.net/distributions/" + .filename + ".asc",
```
Comment (tianon) [inline update.sh:61]: We should probably pull this URL change into a separate PR (although I don't have time right at this moment).

For context, these are the URLs listed on https://www.php.net/downloads (and the URLs Homebrew is using), so they seem more appropriate/correct than these older URLs we've been using.

**Comment by smola** [comment]
> @tianon This is currently failing due to https://bugs.php.net/bug.php?id=79445, but I wanted to get it open to have a place to record the link to that bug.

Adding some retries does the job: `--retry 3 - retry-delay 5 --connect-timeout 10`. 
See: https://github.com/smola/docker-php/actions/runs/181799744

**Comment by tianon** [comment]
Oh snap, it's fully green (90 checks, the full suite twice!)
